examples/xdp_drop_count.rs

```diff
@@ -0,0 +1,110 @@
+extern crate bcc;
+
+use bcc::{
+    BPFBuilder, BccError,
+    XDPMode::{XDP_FLAGS_HW_MODE, XDP_FLAGS_SKB_MODE},
+    XDP,
+};
+use byteorder::{LittleEndian, ReadBytesExt};
+use clap::{App, Arg};
+use std::collections::{hash_map::Entry, HashMap};
```

Comment:
Let's avoid this use declaration syntax, break this up into two separate use declarations.

Reply:
Done!